### PR TITLE
Feature/dcwc 1184 add error pages

### PR DESCRIPTION
**Pull request by nabil386**
### Description

List of proposed changes:

- add error page 400 http response code
- add error page for 503 http response code

### CODE REVIEW

File: pages/400.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import Link from 'next/link'
+
+export default function Error400() {
+  return (
+    <div>
+      <h1>400 Error - Bad request error occurred</h1>
+      <Link href="/dashboard">
+        <a className="text-sm hover:text-bright-blue-solid visited:text-purple-solid underline">
+          Return to dashboard main page
```
Comment (amirpourpanahi) [inline pages/400.js:9]: I think it's better to move it to locales/en.js file

File: pages/503.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export default function Error503() {
+  return <h1>503 - Service unavailable error occurred</h1>
```
Comment (amirpourpanahi) [inline pages/503.js:2]: Same here!

File: pages/400.js (hunk 1 above)
Comment (nabil386) [reply pages/400.js]: done in b18b6c06469fc4da56ab4cde1926e2e1275d6e73

File: pages/503.js (hunk 2 above)
Comment (nabil386) [reply pages/503.js]: done in b18b6c06469fc4da56ab4cde1926e2e1275d6e73

**Review by s-laugh — COMMENTED** [review]
I don't think these pages will work the way you expect them too without proper routing in place... I'd suggest looking at https://nextjs.org/docs/advanced-features/custom-error-page

**Comment by DaceyTom2** [comment]
The error pages seem to work fine without any custom routing or custom error page. Other than the minor A11y issue the PR looks good

**Review by DaceyTom2 — APPROVED** [review]

**Review by amirpourpanahi — APPROVED** [review]

**Pull request merged**